src/index.ts

```diff
@@ -0,0 +1,26 @@
+console.log("Try npm run lint/fix!");
+
+const longString = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer ut aliquet diam.';
+
+const trailing = 'Semicolon'
+
+			const why={am:'I tabbed?'};
+
+const iWish = "I didn't have a trailing space..."; 
+
+const sicilian = true;;
+
+const vizzini = (!!sicilian) ? !!!sicilian : sicilian;
+
+const re = /foo   bar/;
+
+export function doSomeStuff(withThis: string, andThat: string, andThose: string[]) {
+    //function on one line
```

Comment:
I think the implementation for outputting 10 sentences is missing here :)

Reply:
I do not remember creating this file, all the stuff in here I did not create: in the slug.ts file the 10 sentences are created by calling the function like this, console.log(generateRandomSlugs(10));

Reply:
You are supposed to create the script "slug" that outputs the 10 random sentences...